src/main/java/io/cdap/plugin/successfactors/common/util/SuccessFactorsDataTypes.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.successfactors.common.util;
+
+/**
+ * SAP SuccessFactors Data Types Constants File
+ */
+public final class SuccessFactorsDataTypes {
+
+  public static final String SBYTE = "SByte";
+  public static final String BYTE = "Byte";
+  public static final String INT16 = "Int16";
+  public static final String INT32 = "Int32";
+  public static final String INT64 = "Int64";
+  public static final String DOUBLE = "Double";
+  public static final String FLOAT = "Float";
+  public static final String DECIMAL = "Decimal";
+  public static final String GUID = "Guid";
```

Comment:
are we handling this type?

Reply:
No. Removed. 444c10c0add0efdb6799cd258c03b10d1f479f5e